checker/raw_result.go

```diff
@@ -340,6 +340,7 @@ type ArchivedStatus struct {
 // File represents a file.
 type File struct {
 	Path      string
+	LocalPath string
```

Comment:
As Spencer suggested in chat, we can add the localdir to the checker.RawResults.Metadata.Metadata (a `map[string]string`) instead. This way we don't need to modify the `File` struct, and can instead just store it as one more key in the map.

Then, in `Run()`, we just do `localdir := raw.Metadata.Metadata["localdir"]`.

Reply:
It'd be best to find a single place that defines the `localdir`, outside of the check, so that it's also available for other checks that might need it in the future.

Reply:
Done!